runtime/tests/checker/interface_test.go

```diff
@@ -1991,3 +1998,117 @@ func TestCheckInvalidInterfaceUseAsTypeSuggestion(t *testing.T) {
 		errs[0].(*sema.InvalidInterfaceTypeError).ExpectedType,
 	)
 }
+
+//MultipleInterfaceDefaultImplementation
+func TestCheckInvalidMultipleInterfaceDefaultImplementation(t *testing.T) {
+
+	t.Parallel()
+
+	_, err := ParseAndCheck(t, `
+	struct interface InterfaceA {
```

Comment:
Use spaces instead of tabs in code literals. Maybe just do a search-and-replace for tabs and replace them with 4 spaces, then reformat using gofmt.